engine/apps/slack/migrations/0006_rename_channel_id_slackmessage__channel_id_and_more.py

```diff
@@ -0,0 +1,61 @@
+# NOTE: the foreign key checks are disabled and re-enabled in this migration
+# see the following resources as to why
+#
+# https://arc.net/l/quote/hgqztayk
```

Comment:
`https://arc.net/l/quote/...` TIL 😎

Reply:
YIL (yesterday-I-learned) as well 😄:

<img width="373" alt="Screenshot 2024-11-26 at 5 22 52 AM" src="https://github.com/user-attachments/assets/1e5acd4f-1d33-40fe-9c3d-30443bce2520">

"Copy Link to Highlight" in Arc seems to have its own fancy "URL shortener"